resources/quality/abax_pri3/apri3_fast.inst.cfg

```diff
@@ -0,0 +1,14 @@
+[general]
+version = 2
+name = Fast
+definition = abax_pri3
+
+[metadata]
+setting_version = 4
+type = quality
+quality_type = fast
+weight = -1
+global_quality = True
+
+[values]
+layer_height = 0.2
```

Comment:
You added new quality profiles as part of this refactor?

Reply:
This may not be necessary to add. We found that many printers have incorrect quality profiles such as this one: it defines layer_hieght in its extruder qualities. I agree that adding this may break something, @jackha @Ghostkeeper maybe revert this part?

Reply:
Reverted in 6d97fd8dc